src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/help/Get-AzureRmRecoveryServicesAsrNetworkMapping.md

```diff
@@ -33,20 +33,38 @@ The **Get-AzureRmRecoveryServicesAsrNetworkMapping** cmdlet gets information abo
 ```
 PS C:\> $Networkmappings = Get-AzureRmRecoveryServicesAsrNetworkMapping -Network $Network
 ```
+### Example 2
+```
+PS C:\> $Networkmappings = Get-AzureRmRecoveryServicesAsrNetworkMapping -Name $networkMappingName -PrimaryFabric $primaryFabric
+```
 
 Gets all networks mappings for the passed Network.
```

Comment:
@viverm this description was moved from the first example to this one, so the first example no longer has a description. Please add the description back for the first example. Also, it would be nice to show the user where you are getting the value for `$primaryFabric`

Reply:
done